Sight_++_Solution/Intel_Sight_PP/src/classification_result.hpp

```diff
@@ -6,16 +6,18 @@
 struct point {
 	double x;
 	double y;
+
+	point(const int x, const int y) : x(x), y(y) {}
 };
 
 struct ClassificationItem
 {
 	std::string name;
 	double distance;
-	point topLeft;
```

Comment:
Rectangle needs to be the bottom right top left as (0,0) coordinate is top left corner.

Reply:
Are you sure on the coordinate start point?
This is basically a straight-copy taken from the [rs-dnn example](https://github.com/IntelRealSense/librealsense/blob/master/wrappers/opencv/dnn/rs-dnn.cpp).

Reply:
Eh, lets just see what happens then since they do it the normal way